CometServer/Services/Operations/ModelCreatorManager.cs

```diff
@@ -177,12 +177,26 @@ public void CreateEngineeringModelSetupFromSource(Guid source, EngineeringModelS
                 modelRdlToCreate.ShortName = $"{setupToCreate.ShortName}MRDL";
             }
 
-            foreach (var iterationSetup in this.originalToCopyMap.Keys.OfType<IterationSetup>().OrderBy(x => x.IterationNumber))
+            var activeIterationSetup =
```

Comment:
is the solution now that we only support the active iteration and forget about the rest?

Reply:
Yes, because that is what Martin wanted.